geoh5py/objects/object_base.py

```diff
@@ -547,6 +547,17 @@ def remove_children_values(
                 if clear_cache:
                     clear_array_attributes(child)
 
+    def remove_property_group(self, property_group: PropertyGroup):
+        """
+        Remove a property group from the object.
+
+        :param property_group: The property group to remove.
+        """
+        if self._property_groups is None or property_group not in self._property_groups:
```

Comment:
Use `if` block to call/not call the .remove instead of returning None.

Reply:
good call